src/tools/rust-analyzer/crates/syntax/Cargo.toml

```
@@ -17,6 +17,7 @@ either.workspace = true
 itertools.workspace = true
 rowan = "=0.15.15"
 rustc-hash.workspace = true
+rustc-literal-escaper = "0.0.2"
```

Maybe to it to rust-analyzer workspace, like the other dependencies.
```suggestion
rustc-literal-escaper.workspace = true
```